v3/src/components/graph/adornments/count/count-adornment-component.tsx

```diff
@@ -21,6 +21,8 @@ export const CountAdornment = observer(function CountAdornment({model, cellKey}:
   const casesInPlot = dataConfig?.subPlotCases(cellKey)?.length ?? 0
   const percent = model.percentValue(casesInPlot, cellKey, dataConfig)
   const displayPercent = model.showCount ? ` (${percentString(percent)})` : percentString(percent)
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const rerenderOnCasesChange = dataConfig?.casesChangeCount
```

Comment:
Instead of having this unused variable, would it be at all better to do something like add a ref...

`const displayCount = useRef(casesInPlot)`

...and then have a `useEffect` like the below?

```
  useEffect(function handleCaseChanges() {
    return mstReaction(
      () => dataConfig?.casesChangeCount,
      () => displayCount.current = dataConfig?.subPlotCases(cellKey)?.length ?? 0,
      {name: "CountAdornment.handleCaseChanges"}, model)
  }, [cellKey, dataConfig, dataConfig?.casesChangeCount, model])
```
I tried it out and that seems to work fine. I'm not sure it's much better than having the unused variable, though. It's certainly a lot more code.

Reply:
I thought about that, but in addition to being more code it's also a lot more overhead of spinning up reactions and effects. The fact that the component is declared as an `observer` is automatically setting up the reaction we need. All that's necessary is that we reference an observable property within the component, hence the unused variable.